expression/aggregation.go

```diff
@@ -31,17 +31,26 @@ type AggregationFunction interface {
 	// Update during executing.
 	Update(row []types.Datum, groupKey []byte, ctx context.Context) error
 
+	// StreamUpdate updates data using streaming algo.
+	StreamUpdate(row []types.Datum, groupKey []byte, ctx context.Context) (bool, error)
```

Comment:
But how to get the bool value from Update() function to notice the outside world to get a row ?